.github/workflows/new-environment.yml

```diff
@@ -33,6 +33,8 @@ jobs:
     env:
       AWS_ACCESS_KEY_ID:  ${{ secrets.PRIVILEGED_AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY:  ${{ secrets.PRIVILEGED_AWS_SECRET_ACCESS_KEY }}
+      TF_VAR_github_token: ${{ secrets.TERRAFORM_GITHUB_TOKEN }}
```

Comment:
Is this being used anywhere? I can't seem to find the usage. Same for the below.

Reply:
It's a built in Terraform feature, any environmental variable which starts TV_VAR_ will be picked up by Terraform as a variable - https://www.terraform.io/cli/config/environment-variables#tf_var_name